cpu/nrf5x_common/Makefile.dep

```diff
@@ -1,7 +1,3 @@
-ifneq (,$(filter periph_rtc,$(USEMODULE)))
-  USEMODULE += rtt_rtc
-endif
```

Comment:
We could at least have left this in place, it's only used if `periph_rtc` is explicitly requested.

Reply:
It makes no sense if the board doesn't provide that feature.